addons/project/static/src/views/project_task_kanban/project_task_kanban_renderer.js

```diff
@@ -12,30 +12,12 @@ export class ProjectTaskKanbanRenderer extends KanbanRenderer {
 
     }
 
-    get canMoveRecords() {
-        let canMoveRecords = super.canMoveRecords;
-        if (!canMoveRecords && this.canResequenceRecords && this.props.list.isGroupedByPersonalStages) {
-            const { groupByField } = this.props.list;
-            canMoveRecords = groupByField.readonly !== "True";
-        }
-        return canMoveRecords;
-    }
-
-    get canResequenceGroups() {
-        let canResequenceGroups = super.canResequenceGroups;
-        if (!canResequenceGroups && this.props.list.isGroupedByPersonalStages) {
-            const { groupsDraggable } = this.props.archInfo;
-            canResequenceGroups = groupsDraggable && groupsDraggable.readonly !== "True";
-        }
-        return canResequenceGroups;
-    }
-
     canCreateGroup() {
-        return (super.canCreateGroup() && this.isProjectTasksContext() && this.props.list.isGroupedByStage) || this.props.list.isGroupedByPersonalStages;
+        return super.canCreateGroup() && (this.isProjectTasksContext() == this.props.list.isGroupedByStage);// || this.props.list.isGroupedByPersonalStages;--> why is it needed ???
```

Comment:
> My guess is that it was for when you were grouping by personal stages in All Tasks, but I guess it's not necessary anymore?

I forgot to address this comment. This function is still needed to remove the possibility to create stages (other than personal stages) in the My Tasks view (because no possibility to associate them with a project in this view). I've added a comment to clarify.